pkg/bindings/images/build.go

```diff
@@ -143,7 +146,15 @@ func Build(ctx context.Context, containerFiles []string, options entities.BuildO
 		return nil, err
 	}
 	defer tarfile.Close()
-	params.Set("dockerfile", filepath.Base(containerFiles[0]))
+
+	containerFile, _ := filepath.Abs(entries[0])
+	contextDir, _ := filepath.Abs(entries[1])
```

Comment:
need handle the err

Reply:
Ok, I will handle it. Also I found that If Dockerfile/Containerfile is symlink outside context dir it could not be sended. I will look for that. Also podman adds symlinks inside context dir into container, however podman-remote not. I will fix this too.

Reply:
the none followed symlink structure is like that: 
![image](https://user-images.githubusercontent.com/1861031/103164506-324b9e00-481d-11eb-9de2-898727cf1d08.png)

hello3 is missing when building with podman-remote

Reply:
nice job